frappe/patches/v14_0/set_default_is_deleted_for_workflow.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2021, Frappe Technologies Pvt. Ltd. and Contributors
+# MIT License. See license.txt
+
+from __future__ import unicode_literals
```

Comment:
```suggestion
# Copyright (c) 2022, Frappe Technologies Pvt. Ltd. and Contributors
# MIT License. See LICENSE

```

Is there a particular reason for adding this, or is this a part of some template? We're not supporting PY2 on v14 AFAIK so we can do without it.


---

btw, is this patch really required?

<img width="725" alt="Screenshot 2022-02-09 at 11 00 15 AM" src="https://user-images.githubusercontent.com/36654812/153127859-3ffaed6b-3a7b-4097-a665-d6b6b9fd18be.png">

If Frappe doesn't handle this correctly, there might have to be a fix in `DBTable.alter` instead.